libcaf_core/caf/actor_system_config.hpp

```diff
@@ -252,6 +252,13 @@ class actor_system_config {
   size_t work_stealing_relaxed_steal_interval;
   size_t work_stealing_relaxed_sleep_duration_us;
 
+  // -- config parameters for the logger ---------------------------------------
+
+  std::string logger_filename;
+  std::string logger_verbosity;
+  bool logger_console;
```

Comment:
These two bools team up to model three possible states, which seems a bit fiddly to me. We could again use atoms: `console = 'NONE' | 'COLORED' | 'UNCOLORED'`, whereas `'NONE'` is the default.